_core/smart/overview.md

```diff
@@ -1,135 +1,55 @@
 ---
 layout: core
-description: "Blockstack smart contracting language"
+description: "Blockstack Clarity: Introduction"
 permalink: /:collection/:path.html
 ---
-# Welcome to Clarity
+# Introduction to Clarity
 {:.no_toc}
 
-Clarity is Blockstack's smart contracting language for use with the Stacks blockchain. Clarity supports programmatic control over digital assets within the Stacks blockchain (for example, BNS names, Stacks tokens, and so forth). This section discusses the following topics:
+Clarity is a smart contracting language for use with the Stacks 2.0 Blockchain. It supports programmatic control over digital assets.
 
 * TOC
 {:toc}
 
-<div class="uk-card uk-card-default uk-card-body">
-<h5>Clarity is in pre-release</h5>
-<p>Clarity, its accompanying toolset, and the SDK are in pre-release.  If you encounter issues with or have feature requests regarding Clarity, please create an issue on the <a href='https://github.com/blockstack/blockstack-core/issues' target='_blank'>blockstack/blockstack-core</a> repository. To read previous or join ongoing discussions about smart contracts in general and Clarity in particular, visit the <strong><a href='https://forum.blockstack.org/c/clarity' target='_blank'>Smart Contracts</a></strong> topic in the Blockstack Forum.
-</p>
-</div>
+## Smart contracts
 
-## Who should use smart contracts?
+Smart contracts allow two parties to exchange anything of value (money, property, shares), in an automated, auditable, and secure way _without the services of a middleman_. Nick Szabo introduced the canonical metaphor for smart contracts, a vending machine.
```

Comment:
I'd be in favor of a more generic definition of a smart contract here. They enable much more than just exchanging things of value, and could be introduced in that way. Something like:


Smart contracts are programs that can be executed in a transparent and verifiable fashion. When deployed to a blockchain, they allow parties to submit transactions to the blockchain which execute those programs. Such smart contract operation is automated, auditable, and consistent.

Smart contracts enable parties to exchange things of value in an automated and secure way. For example, a "swap contract" could allow two parties to exchange two different assets without trusting one another. One party would initiate the swap by submitting a transaction to transfer an asset to the contract. Then, when the second party transferred the second asset, the smart contract would distribute the assets to each party. If the second party never transferred the second asset, the smart contract would release the first asset back to the first part.

Reply:
Love it

Reply:
I was going back and forth about this example as well.

Do you think there's a way we can frame smart contracts as a tool for enabling a user-owned internet?  I think we're doing ourselves a disservice trying to talk about smart contracts in terms of programmable money (and nothing else).  An example I was toying with last night looks like:

`Smart contracts are programs that run on a blockchain.  In Blockstack, the main use-case for smart contracts is to manage _global shared state_ for an app.  This is data that is collectively owned and curated by all of an app's users.  For example, the list of all subreddits is global shared state in Reddit.  As another example, the catalog of movies is global shared state in YouTube.  Every user can see and use this data when the use the app, and every user can propose changes to this data, but the data itself belongs to everyone.`

`Smart contracts encode and enforce the rules for curating this data, and are often used to ensure that users can exchange valuable data in an automated and secure way.  For example, a smart contract could allow anyone to add a new movie to a movie catalog, but require that someone first pay for a decryption key before anyone can view it.  To do this, the smart contract could be written such that the movie creator had to publicly disclose the decryption key in order to receive payment.  Such a system would allow movie creators to make money by adding movies to the movie catalog that people want to watch.`

`Because smart contracts run on top of a blockchain, anyone can query them, and anyone can submit transactions to run their code.  The blockchain stores the history of all accepted transactions, so anyone can audit the blockchain in order to independently verify that an app's global shared state has been managed correctly according to the smart contract's rules.`

Reply:
Even better!